compiler/dyno/tools/chpldoc/chpldoc.cpp

```diff
@@ -1459,6 +1605,12 @@ static const owned<RstResult>& rstDoc(Context* context, ID id) {
   return QUERY_END(result);
 }
 
+
+/*
+  Query to get the module path for a given id. Path may be just the name of
+  the module in the case of a top-level module, or it may contain the parent(s)
+  of the module if it is a submodule.
```

Comment:
In the follow-up PR, it would be nice to update this comment to say it's talking about the module's symbol path an not a file path.